### PR TITLE
Cyborg Model Advertisement Update, or How I Stopped Having To Guess Your Model

### PR DESCRIPTION
**Pull request by GoldenAlpharex**
## About The Pull Request
Have you ever looked at a borg and shift-clicked to figure out what kind of borg they were, and subsequently went like "fuck, how do I know which model they are???" as you noticed that nothing was indicating their model in there?

Well, boy do I have the solution for you.

### Introducing: Cyborg Models In Examine
image

It's simple, it's slick, it's incredibly convenient and useful!

It even properly switches between `a` and `an` accordingly! (BYOND text macros my beloved)
image

It also works with the borg chameleon module, which means it won't give out that you're a syndicate saboteur!
image
image

## Why It's Good For The Game
Having players rely on just the sprite to figure out what model a cyborg is isn't very great for accessibility, especially for models that also have different color scheme variants, such as the syndicate borgs. We show a lot of information on humans over examine, I was surprised nobody ever has ever thought of adding it to cyborgs yet. I guess it counts as consistency too, since, y'know, you'd expect to be able to tell what model they are from examining them.

## Changelog

:cl: GoldenAlpharex
qol: After several incidents related to miscommunications regarding robot models, Nanotrasen's Robotic Appearance Normalization Department has decided to make it extra-obvious which model each cyborg is - Just give them a closer look, it's that easy!
/:cl:

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/examine.dm [hunk 1]
```diff
@@ -3,6 +3,9 @@
 	if(desc)
 		. += "[desc]"
 
+	var/model_name = model ? "\improper [model.name]" : "\improper Default"
```
Comment (Fikou) [inline code/modules/mob/living/silicon/robot/examine.dm:6]: why do you need the improper

Comment (GoldenAlpharex) [reply code/modules/mob/living/silicon/robot/examine.dm:6]: The improper is for the usage of the `\an` macro, otherwise it just displays nothing.

**Comment by ATH1909** [comment]
I forget, does the cardborg disguise normally give you a fake description, or do we not need to account for it here?

**Comment by 13spacemen** [comment]
rather than "Engineering" it should be <b>Engineering</b> so that you can make it out at a glance

File: code/modules/mob/living/silicon/robot/examine.dm [hunk 2]
```diff
@@ -3,6 +3,9 @@
...
+	. += "\nIt is currently \a \"[model_name]\"-type cyborg.\n"
```
Comment (ShizCalev) [inline code/modules/mob/living/silicon/robot/examine.dm:7]:
```suggestion
	. += "\nIt is currently \a \"[span_bold(model_name)]\"-type cyborg.\n"
```

File: code/modules/mob/living/silicon/robot/examine.dm [hunk 3]
```diff
@@ -3,6 +3,9 @@
 	if(desc)
 		. += "[desc]"
 
+	var/model_name = model ? "\improper [model.name]" : "\improper Default"
+	. += "\nIt is currently \a \"[span_bold(model_name)]\"-type cyborg.\n"
```
Comment (ShizCalev) [inline code/modules/mob/living/silicon/robot/examine.dm:7]:
```suggestion
	. += "\nIt is currently \a \"[span_bold("[model_name]")]\"-type cyborg.\n"
```

**Pull request merged**